css/legacy/includes/_impact.scss

```diff
@@ -608,3 +608,31 @@ i.fa-impact-manipulation {
 #impact_settings {
    display: none;
 }
+
+.cy-node-title {
+   text-align: center;
+   //if valignBox : top
+   //margin-top: 3em;
+   //margin-bottom: 3em;
+   //padding-top: 3em;
+   //padding-bottom: 3em;
+   //if valignBox : bottom
+   margin-top: 0.5em;
+   margin-bottom: 3em;
+   padding-top: 0.5em;
+   padding-bottom: 3em;
+   line-height: 1.1em;
+}
+
+.cy-node-label {
+   font-size: 12px;
+   color: #666;
+   line-height: 1.1em;
+   margin-bottom: 1.1em;
+}
+
+.cy-node-comment {
+   font-size: 10px;
+   color: #999;
```

Comment:
I think we should try to reuse values already defined by tabler if they exist.

Here we could use the standard `text-muted` class instead of defining a custom color.